### ISSUE TITLE
oversample_thin reports acceptance rate 1

### ISSUE THREAD

**Issue by cmbant** [CobayaSampler/cobaya#310]
As in example yaml from https://github.com/CobayaSampler/cobaya/issues/308.

```
 All chains are ready to check convergence and learn a new proposal covmat
[0 : mcmc]  - Acceptance rate: 1.000 = avg([1.0, 1.0, 1.0, 1.0])
[0 : mcmc]  - Convergence of means: R-1 = 0.011349 after 333 accepted steps = sum([80, 86, 86, 81])
```

**Comment by vivianmiranda** [comment]
It is not exactly one but still weird in many of my LSST chains (it seem to slowly drifts toward 1)

I attached an example:

[acceptance_rate_lssty1_cosmicshear.txt](https://github.com/CobayaSampler/cobaya/files/12368891/acceptance_rate_lssty1_cosmicshear.txt)

**Comment by cmbant** [comment]
I think it makes sense, but the message is rather misleading - these numbers are basically the inverse of the mean thinned weight of the samples remaining after thinning. Doesn't look as though there's a very simple way to get the actual acceptance rate, so I've changed the reporting message to be consistent:
https://github.com/CobayaSampler/cobaya/pull/319

**Comment by JesusTorrado** [comment]
Addressed by #319
